### DISCUSSION TITLE
Use pre-installed server instead of the tool downloading it

### DISCUSSION THREAD

**Discussion by tnaeg** [williamboman/nvim-lsp-installer#517] [Q&A] [answered]
I have installed clangd for other reasons so looking to have it use the same config I have set up for attachment to other types of servers
where I have found good use of the nvim-lsp-installer. 

I have tried som modifications to the install script but my nvim->lua is not yet there.

**Comment by williamboman** [comment, marked as answer]
Hello! I'm afraid I'm not following you. Do you mean you want to manage some servers outside of nvim-lsp-installer? In that case you could just set them up manually in your config, for example via lspconfig:

```lua
lspconfig.clangd.setup {}
```

**Reply by tnaeg** [reply]
So I run something similar to 
```
lsp_installer.on_server_ready(function(server)                                       
                                                         
if server.name == "clangd" then                          
  -- Load additional configs                  
end                                                      
                                                         
if server.name == "pyright" then                     
 -- Load specific configs
end          
```                                            
But if you use the the `lspconfig.clangd.setup()` I would have to write this manually for the clangd parts
Wanted to see if you hade any idea to have the install part of `nvim-lsp-installer` check if clangd exists in `/usr/bin/` instead of installing it from github

**Reply by williamboman** [reply]
Ah. No, nvim-lsp-installer won't check for any existing installations.
